### PR TITLE
Base repository

### PR DESCRIPTION
**Pull request by adamklingbaum**
- Adds webpack, linter, template src code, and dist server

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: path.join(__dirname, 'src', 'index.js'),
+  output: {
+    path: path.resolve(__dirname, 'dist'),
```
Comment (oliviaschwerdt) [inline webpack.config.js:7]: If we add the `filename` property, would this allow us to pre-populate `src/index.html` with the corresponding script tag? `<script type="text/javascript" src="main.js"></script>`

Just suggestion, please leave as-is if you think that's a better approach!
```suggestion
    filename: 'main.js',
```

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>betrothed</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (oliviaschwerdt) [inline src/index.html:9]: This suggestion is dependent on the comment I left on `webpack.config.js`. Just suggestion, please leave as-is if you think that's a better approach!
```suggestion
    <script type="text/javascript" src="main.js"></script>
```

**Review by oliviaschwerdt — APPROVED** [review]
Looks good, Adam! Thanks for getting the client codebase set up for us! I left a couple comments, but approving since they are suggestions and aren't necessary to move forward.

File: webpack.config.js (hunk 1 above)
Comment (hithisiskhai) [reply webpack.config.js:7]: I think that this is a good idea. Will add to talking points when backend team reconvenes. Thank you Olivia!

**Review by hithisiskhai — APPROVED** [review]

File: src/index.html (hunk 2 above)
Comment (hithisiskhai) [reply src/index.html:9]: Noted in previous comment

File: webpack.config.js (hunk 1 above)
Comment (adamklingbaum) [reply webpack.config.js:7]: Olivia/Khai: this is not needed. The `HtmlWebpackPlugin` I included on line 32 already adds the bundle script to `index.html`.

Here is an example of the resulting `index.html` from the webpack build:
Screen Shot 2021-12-09 at 4 49 25 PM

File: src/index.html (hunk 2 above)
Comment (adamklingbaum) [reply src/index.html:9]: See my response to your comment on `webpack.config.js`

**Pull request merged**